app/test.scss

```diff
@@ -18,8 +18,23 @@
         width: 150px;
 
         .form-row {
-            padding: 5px 10px;
+            padding: 0 10px;
             display: block;
+            margin-bottom: 6px;
+        }
+        input, select {
+            width: 100%;
+            box-sizing: content-box;
+        }
+        input[type='checkbox'] {
+            width: auto;
+        }
+        button {
+            border-radius: 4px;
+            background: #dedede;
+            border: 1px solid #aaa;
```

Comment:
Is it OK to use hex colors directly instead of Sass variables?

Reply:
@mahdiaryayi-fs actually this is test file which we use it for developer test and no more. we use to break rules in this file from a long time ago ;)